app/components/single-elements/ArticleTitle.tsx

```
@@ -13,25 +58,25 @@ export const ArticleTitle = ({
 
   if (link) {
-    const urlLink = DOMPurify.sanitize(`<a href="${link}" target="_blank" class="is-primary">${tag}</a>`);
-    
-    return (
-      <div dangerouslySetInnerHTML={{ __html: urlLink }} />  
-    )
+    const urlLink = DOMPurify.sanitize(
+      `<a href="${link}" target="_blank" class="is-primary">${tag}</a>`
+    );
+
+    return <div dangerouslySetInnerHTML={{ __html: urlLink }} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)